spectr/changes/add-uneditable-comment-ranges/specs/comment-syntax-md/spec.md

```
@@ -0,0 +1,283 @@
+- **AND** a protected range from line 5 to line 7 SHALL be created
+
+#### Scenario: Marker before and after code fence
+
+- **GIVEN** a Markdown file with:
+  ```markdown
+  <!-- conclaude-uneditable:start -->
+  ```javascript
+  // This is code
+  ```
+  <!-- conclaude-uneditable:end -->
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)